### DISCUSSION TITLE
Docusaurus DocSearch 搭建 - Alan|前端博客

### DISCUSSION THREAD

**Discussion by giscus** [3Alan/site#8] [Announcements]
# Docusaurus DocSearch 搭建 - Alan|前端博客

通过 Github Actions 运行 Algolia DocSearch 实现 Docusaurus 搜索功能

https://www.alanwang.site/posts/%E5%8D%9A%E5%AE%A2%E6%90%AD%E5%BB%BA%E7%B3%BB%E5%88%97/Docusaurus-%E6%90%9C%E7%B4%A2

**Comment by Shake-Jarjane** [comment]
博主你好，按照你的方法，我用提交代码触发时是正常工作的，但是用deployment 触发时好像没反应，我的是通过github action部署在github pages上的

**Reply by Shake-Jarjane** [reply]
没事了，解决了😂
